cli/src/ios/update.ts

```diff
@@ -158,7 +163,7 @@ end`,
 async function generatePodFile(
   config: Config,
   plugins: Plugin[],
-): Promise<string> {
+): Promise<{ dependenciesContent: string; relativeCapacitoriOSPath: string }> {
```

Comment:
better create a function for getting the relativeCapacitoriOSPath and use it where needed, this function should not return it